### PR TITLE
Update README for cargo-screeps 0.2.0

### PR DESCRIPTION
**Pull request by daboross**

### CODE REVIEW

**Comment by ASalvail** [comment]
Don't forget https://github.com/daboross/screeps-in-rust-via-wasm/tree/master/cargo-screeps

**Comment by daboross** [comment]
huh, I'd completely forgotten, you're right.

I just did a search for `cargo screeps` and completely forgot the options are documented separately.

File: cargo-screeps/README.md [hunk 1]
```diff
@@ -13,25 +13,38 @@ This is not fully tested, but feel free to use! Issues are welcome.
...
 2. read `target/*.js` and `target/*.wasm`, keeping track of filenames
 3. read `screeps.toml` for upload options
 4. upload all read files to server, using filenames as the filenames on the server.
 
-### `--check`:
+### `copy`:
 
-1. run `cargo web --check --target=wasm32-unknown-unknown`
+1. run build.
+2. copy `target/main.js` and `target/compiled.wasm` to `<destination directory>/<branch name>/`
```
Comment (ASalvail) [inline cargo-screeps/README.md]: You are assuming that you are keeping the default names. Was that intentional?

Comment (daboross) [reply cargo-screeps/README.md]: I'll add a note mentioning that! It was mostly lazyness.

**Pull request merged**